### PR TITLE
solved

### PR DESCRIPTION
**Pull request by iamfortune**

### CODE REVIEW

File: daily-coding-challenges/26-05-2020/findGreatestNumInArray.js [hunk 1]
```diff
@@ -4,4 +4,11 @@
 */
 const findGreatestNumInArray = (arr) => {
 	/* Your implementation goes here */
-}
\ No newline at end of file
+	let maxNum = arr[0];
```
Comment (meekg33k) [inline daily-coding-challenges/26-05-2020/findGreatestNumInArray.js:11]: @iamfortune, what happens if `arr` is `null` or `undefined`? Is there a way you could rewrite this to handle the edge case?

**Review by meekg33k — CHANGES_REQUESTED** [review]
Good start @iamfortune, I've added some comments about handling edge cases. Please read through and and let me know if you have any questions

File: daily-coding-challenges/26-05-2020/findGreatestNumInArray.js (hunk 1 above)
Comment (iamfortune) [reply daily-coding-challenges/26-05-2020/findGreatestNumInArray.js:11]: Sure, going to try sir

Comment (meekg33k) [reply daily-coding-challenges/26-05-2020/findGreatestNumInArray.js:11]: Great @iamfortune. I hope you understand what I'm asking to do here. Please let me know if you don't or if you have any questions

Comment (iamfortune) [reply daily-coding-challenges/26-05-2020/findGreatestNumInArray.js:11]: you want me to write the algorithm in a way that it recognizes and probably returns -1 or wouldn't work if array is null (empty) or not defined? yeah?

File: daily-coding-challenges/26-05-2020/findGreatestNumInArray.js [hunk 2]
```diff
@@ -4,6 +4,7 @@
 */
 const findGreatestNumInArray = (arr) => {
 	/* Your implementation goes here */
+	if (arr.legnth ==1) {return arr[0]}
```
Comment (meekg33k) [inline daily-coding-challenges/26-05-2020/findGreatestNumInArray.js]: @iamfortune, I like that you worked on this and came up with a solution. This is good but here are some issues you may have to fix:
- I see a typo, can you correct the text from `legnth` to `length`? 
- Adding this line allows us check if the array has at least one element. What happens if the array is `undefined` or `null`?

**Review by iamfortune — COMMENTED** [review]
stuck on getting the null and undefined input working...send help 

still trying to find my way around it

**Comment by meekg33k** [comment]
Alright @iamfortune, good job with being consistent so far. Let's talk about it when we meet tomorrow.

File: daily-coding-challenges/26-05-2020/findGreatestNumInArray.js (hunk 2 above)
Comment (iamfortune) [reply daily-coding-challenges/26-05-2020/findGreatestNumInArray.js]: done this, but somehow i couldn't seem to get the edge case for null and undefined to work!>/
will gladly accept help on this front

**Pull request merged**